packages/interaction-kit/src/commands/slash-command.ts

```diff
@@ -9,6 +9,10 @@ type CommandArgs = {
 	description: string;
 	defaultPermission?: boolean;
 	options?: Input[];
+	onAutocomplete?: (
+		interaction: SlashCommandAutocompleteInteraction,
+		application: Application
+	) => unknown;
```

Comment:
I think both this and the handler should have a return type of void, not unknown actually. This should prevent people from returning anything, since you're not going to do anything with that return value